### DISCUSSION TITLE
Time Speed + Pause to simulation

### DISCUSSION THREAD

**Discussion by delogo11** [boytchev/COVID-19-SIM#50] [Q&A]
Hello,

I have been playing around with your COVID-19 simulator and I really appreciate the effort and all the parameters you have added.

The option to speed up the simulation is a nice addition, but I was wondering if it is possible to pause the simulation to check for the infected people (for example, after running the speed up the simulation for a couple of minutes).

Kind regards,

delogo

**Comment by boytchev** [comment]
Hi -

For the time being there is no option to pause the simulation, but I also think that it will be important to be able to pause the simulation and then to resume it.

Thanks for the suggestion, I'll add it to the TODO list.

- Pavel

**Comment by boytchev** [comment]
Hi -

It is done. It might have some bugs, but the major functionality is there. Clicking on the timer (top right corner) will pause/resume the simulation. While the simulation is paused, you can still navigate in it.

\- Pavel

**Reply by delogo11** [reply]
Hello! 

Thank you for implementing this functionality.

It works, and the number of days that have passed are a nice touch too :).
